docs/src/intro-csharp.md

```diff
@@ -91,7 +91,7 @@ Edit the `UnitTest1.cs` file with the code below to create an example end-to-end
 }>
 <TabItem value="nunit">
 
-```csharp
+```csharp title="UnitTest1.cs"
```

Comment:
I wonder why `UnitTest1.cs`, and not `E2ETest.cs` or just `Test1.cs`?

Reply:
Thats the default `dotnet new` gives you.